Create rule S6650 (#2266)
SonarSource/rspec: rules/S6650/javascript/rule.adoc

== Why is this an issue?

> Renaming imports, exports, or destructuring assignments to the same name makes no sense and should be removed. You may accidentally end up with such code if you do a refactoring and change the local name in several places.

Reviewer: ```suggestion
Renaming imports, exports, or destructuring assignments to the same name is redundant and can be safely removed. You may accidentally end up with such code if you do a refactoring and change the local name in several places.
```

Reviewer: avoid phrases like 'makes no sense' and be more factual ie. 'is redundant'
